packages/ckeditor5-table/tests/manual/table.html

```diff
@@ -201,4 +201,38 @@
 		</tr>
 		</tbody>
 	</table>
+
+	<p>Nested tables</p>
+
+	<table>
+		<tr>
+			<td></td>
+			<td></td>
+			<td></td>
+		</tr>
+		<tr>
+			<td></td>
+			<td>
+				Paragraph before table
```

Comment:
This should be a `<p>`, if we want to have here content that would normally be created by an editor.

Also, all empty `<td>`s should be filled with `&nbsp` too.

Reply:
Done.